src/target/llvm/codegen_llvm.cc

```diff
@@ -184,6 +188,104 @@ void CodeGenLLVM::AddFunctionInternal(const PrimFunc& f, bool ret_void) {
   }
 }
 
+void CodeGenLLVM::LinkParameters(const Map<String, LinkedParam> params) {
+  // It would be nice to de-dupe these declarations frm src/tir/transforms/make_packed_api.cc,
```

Comment:
It is possible to list more into make_packed_api later, if we have intrinsics like

```
value = tir.lookup_linked_param_in_attr(id)

```

The next steps should be supported once we have return value in PackedFunc. cc @ZihengJiang

Reply:
I see